build/config/android/config.gni

```diff
@@ -94,20 +94,19 @@ if (is_android) {
 
   # Subdirectories inside android_ndk_root that contain the sysroot for the
   # associated platform.
-  _android_api_level = 16
+  _android_api_level = 24
   x86_android_sysroot_subdir =
       "platforms/android-${_android_api_level}/arch-x86"
   arm_android_sysroot_subdir =
       "platforms/android-${_android_api_level}/arch-arm"
   mips_android_sysroot_subdir =
       "platforms/android-${_android_api_level}/arch-mips"
-  _android64_api_level = 21
   x86_64_android_sysroot_subdir =
-      "platforms/android-${_android64_api_level}/arch-x86_64"
+      "platforms/android-${_android_api_level}/arch-x86_64"
   arm64_android_sysroot_subdir =
-      "platforms/android-${_android64_api_level}/arch-arm64"
+      "platforms/android-${_android_api_level}/arch-arm64"
   mips64_android_sysroot_subdir =
-      "platforms/android-${_android64_api_level}/arch-mips64"
+      "platforms/android-${_android_api_level}/arch-mips64"
```

Comment:
We don't have any builders that build for MIPS currently. Though it should not be stretch to get one up and running.